locales/ast_ES/LC_MESSAGES/duckduckgo.po

```diff
@@ -4015,6 +4019,9 @@ msgctxt "reasons-to-use-duckduckgo"
 msgid "We don't track you. Ever."
 msgstr "Nun te rastrexamos. Daveres."
 
+msgid "We don't track you."
+msgstr "Nun te rastrexamos."
```

Comment:
Hi @enolp - it looks like there's a duplicate msgid here.

https://github.com/duckduckgo/duckduckgo-locales/blob/39bd96c/locales/ast_ES/LC_MESSAGES/duckduckgo.po#L4015-L4017

If you delete one of these, the tests should pass. Thanks!!

Reply:
Fixed :)